content/docs/0.19.x/reference/miscellaneous/distributor/index.md

```diff
@@ -18,6 +18,11 @@ In most cases, only a subset of these need to be configured.
 After the application is deployed,
 its subscription can be changed directly in the Bridge.
 
+Note that some services use `cd-connectors` or `go-sdk`
```

Comment:
`cd-connectors` -> `cp-connector`

Reply:
done.  Also caught the same typo in the following line.